test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -726,6 +726,30 @@ namespace StackTest {
             & $powershell $switch -noprofile -command "[System.Threading.Thread]::CurrentThread.ApartmentState" | Should -BeExactly $apartment
         }
 
+        It "WPF will work when using -sta" -Skip:(!$IsWindows) {
```

Comment:
I'm afraid this is a slow test. Perhaps there is a faster COM object that we could use.

Reply:
WPF is interesting as users want to be able to create WPF with PowerShell and it requires STA.

Reply:
It probably should be tagged slow.

Reply:
Will tag as Slow (took about 3.2s on my machine).